File: Estrutura_Repetição/JavaScript/Ex1114.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

while(true){
    var senha = Number(prompt())
    var resultado = senha != 2002 ? "Senha Invalida" : "Acesso Permitido"
    console.log(resultado)
    if(resultado == "Acesso Permitido")
    break;
    
}

File: Estrutura_Repetição/Clojure/Ex1114.clj
(defn verificar-senha []
  (let [senha-correta 2002]
    (letfn [(verificar [senha]
              (if (= (Integer/parseInt senha) senha-correta)
                (println "Acesso Permitido")
                (do
                  (println "Senha Invalida")
                  (verificar (read-line)))))]
      (verificar (read-line)))))

(verificar-senha)